contributors.txt

```diff
@@ -0,0 +1,57 @@
+GitExtensions Project Contributors Certification of Origin and Rights
```

Comment:
💭 The project isn't especially consistent on "GitExtensions" vs. "Git Extensions". I assume this is the preferred form?

Reply:
Everywhere we use "Git Extensions", I'll fix this one up
Thank you